site/app/templates/course/CourseMaterials.twig

```diff
@@ -64,12 +64,11 @@
                     <a href="{{ course_material.getUrl }}" onclick="markViewed([{{ course_material.getId }}])" onauxclick="markViewed([{{ course_material.getId }}])">{{ course_material.getUrlTitle }}</a>
                 {% else %}
                     <i class="fas fa-file" style="vertical-align: text-bottom;"></i>
+                    {{ name }}
                 {% endif %}
                 {% set extension = name|split('.')|last|lower %}
                 {% if '.' ~ extension in ['.pdf', '.jpg', '.jpeg', '.c', '.cpp', '.s', '.twig', '.py', '.java', '.png', '.txt', '.h', '.html', '.php', '.js', '.sql', '.sh', '.md', '.csv', '.salsa', '.erl', '.oz', '.pl', '.hs', '.gif'] %}
                     <a class="popout-item" target="_blank" href="{{ display_file_url }}?course_material_id={{ course_material.getId }}" aria-label="Pop up {{ name }} in a new window" tabindex="0" style="text-decoration: none;">{{ name }} <i class="fas fa-window-restore" title="Pop up the file in a new window"></i></a>
```

Comment:
It seems like if we enter this if statement, we will output `{{ name }}` as part of the `a` tag, and so if we hit the `else` above and this `if`, then we output `{{ name }}` twice.

Additionally, having stared at this whole thing, should we tuck all the lines between 69-75 under the `else` branch of the `if course_material.isLink` check?

Reply:
I forgot to push my last commit so now it should be up to date. The name issue no longer occurs. Yeah I'll try to tuck the whole thing in the else to make it a bit better organized and update the PR text.